.travis.yml

```diff
@@ -37,7 +37,7 @@ script:
         cargo rustc --verbose --example ci_test -- -Zno-trans &&
         cargo rustc --verbose --example key_value_store -- -Zno-trans &&
         cargo rustc --verbose -- -Zno-trans &&
-        cargo test  --verbose --features use-mock-crust
+        env RUSTFLAGS="-C codegen-units=8" cargo test --release --verbose --features use-mock-crust --test mock_crust_tests
```

Comment:
We don't want to limit this to only run the mock crust tests do we?  I think ` --test mock_crust_tests` should be removed.